lib/src/core/transaction_signer.dart

```diff
@@ -48,8 +59,8 @@ Future<_SigningInput> _fillMissingData({
             data: transaction.data,
             value: transaction.value,
             gasPrice: gasPrice,
-            maxPriorityFeePerGas: transaction.maxPriorityFeePerGas,
-            maxFeePerGas: transaction.maxFeePerGas,
+            maxPriorityFeePerGas: maxPriorityFeePerGas,
+            maxFeePerGas: maxFeePerGas,
```

Comment:
@redDwarf03 with the changes proposed it will not be possible to set maxFeePerGas manually, right?

Reply:
Only in the case of EIP1559. See https://www.blocknative.com/blog/eip-1559-fees

Reply:
yes, what I mean, it is not possible to override it in case it's needed